contentcuration/contentcuration/assets/shared/views/AppBar.vue

```diff
@@ -0,0 +1,88 @@
+<template>
```

Comment:
This is the App Bar that used to be rendered in the Django template.

I imported it here to reimplement it using Vuetify components.